test/unit/org/apache/cassandra/repair/autorepair/FixedSplitTokenRangeSplitterTest.java

```diff
@@ -61,16 +61,27 @@ public class FixedSplitTokenRangeSplitterTest
     private static final String TABLE1 = "tbl1";
     private static final String TABLE2 = "tbl2";
     private static final String TABLE3 = "tbl3";
+    private static final int totalTokenRanges = 3;
```

Comment:
Don't need to solve this here, but one thing I think would  be nice to solve in our tests is to test where `num_tokens > 1` to really exercise this change, maybe we can do this as a follow on?